python_modules/dagster/dagster_tests/core_tests/serdes_tests/import_readline.py

```diff
@@ -0,0 +1,2 @@
+# This will hang in OSX if it is imported in a subprocess with stdin not equal to DEVNULL
```

Comment:
not sure this is OSX specific - that line of thinking was incorrectly drawn from "local (osx) fails, remote (linux) works" but  the other difference between "local" and "remote" was going through `open_ipc_subprocess` at all